po/de.po

```diff
@@ -1725,9 +1723,8 @@ msgid "failed to parse binding"
 msgstr "Parsen der Tastenbelegung fehlgeschlagen"
 
 #: src/keymap.cpp:1028
-#, fuzzy, c-format
```

Comment:
`c-format` should have been kept. I fixed it in master via cdc6568194bd95696d58b8683c11f63d3551552c.

Reply:
Dang it, that was definitely an uncareful <kbd>d</kbd><kbd>d</kbd> in vim. Thanks for repairing it! <3